### PR TITLE
WNP 129 NA + EU (fixes #14864)

### LINKED ISSUE #14864: WNP 129 NA + EU
Hello. Please refer to the brief doc [here](https://docs.google.com/document/d/1ln1zuCm872pR8VtIJCfY5RFxGr948cdYIf9pm3iAu84/edit) (mozilla internal only)

### PR DESCRIPTION
**Pull request by wen-2018**
## One-line summary

This PR adds WNP 129 for NA + EU.

## Issue / Bugzilla link

https://github.com/mozilla/bedrock/issues/14864

## Testing

NA:
http://localhost:8000/en-US/firefox/129.0/whatsnew/
http://localhost:8000/en-CA/firefox/129.0/whatsnew/

EU:
http://localhost:8000/fr/firefox/129.0/whatsnew/
http://localhost:8000/de/firefox/129.0/whatsnew/
http://localhost:8000/es-ES/firefox/129.0/whatsnew/
http://localhost:8000/en-GB/firefox/129.0/whatsnew/
http://localhost:8000/it/firefox/129.0/whatsnew/
http://localhost:8000/pl/firefox/129.0/whatsnew/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html
Comment (janbrasna) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html:90]: FYI, the localised images in img/l10n, when all named the same, just in different locale folders, can be referenced by the locale automagically via https://bedrock.readthedocs.io/en/latest/coding.html#l10n-img (or in this case ` "l10n": True,` on the `picture` helper as shown at the end of https://bedrock.readthedocs.io/en/latest/coding.html#resp-img …)

Comment (wen-2018) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html:85]: Thanks for the suggestions!

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html
Comment (wen-2018) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html:62]: The SVG is added directly here to make it easy to implement dark mode.

**Comment by wen-2018** [comment]
This is scheduled to launch on Aug 6.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html
Comment (craigcook) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html:70]: Since this doesn't have any content it should probably be a generic `div`

File: media/img/firefox/whatsnew/whatsnew129-eu/qr-code-fr.png
Comment (craigcook) [inline media/img/firefox/whatsnew/whatsnew129-eu/qr-code-fr.png:1]: With the localized images I think this one is redundant? Seems safe to delete.

**Review by craigcook — APPROVED** [review]
Nice work! Only the tiniest of tidy-up suggestions. I leave it to you to address the comments before merging, or don't, it doesn't hurt anything. 😸

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html
Comment (janbrasna) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html:11]:
```suggestion
{% block page_title %}{{ ftl('whatsnew-page-title-v2') }}{% endblock %}
```

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-na.html
Comment (janbrasna) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-na.html:9]:
```suggestion
{% block page_title %}{{ ftl('whatsnew-page-title-v2') }}{% endblock %}
```

**Review by janbrasna — CHANGES_REQUESTED** [review]
One tiny suggestion, this was somewhat forgotten before and fixes to previous WNPs shipped just recently — there are newer strings for the page titles:

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html
Comment (janbrasna) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx129-eu.html]: 👍 yea I was kinda surprised when doing l10n images the first time, somewhat expecting things to end up elsewhere than was the reality;D so thought to share that early on to point out the bits in docs that tackle this;)

File: bedrock/firefox/views.py
Comment (janbrasna) [inline bedrock/firefox/views.py:188]: These seem reformatted without any change?

File: bedrock/firefox/views.py
Comment (janbrasna) [inline bedrock/firefox/views.py:408]: also here is seems unrelated to this PR? the weird thing is not the whole file is reformatted, only some lines(?)

File: bedrock/firefox/views.py
Comment (janbrasna) [inline bedrock/firefox/views.py:851]: these also seem unrelated

File: bedrock/firefox/views.py
Comment (janbrasna) [inline bedrock/firefox/views.py:754]: there are several occurrences of this exact change too

**Review by janbrasna — COMMENTED** [review]
Oh that's lovely, I never noticed this animation before, so cool!

I've noticed something a bit off here, changing around the .py files formatting, that might have been unintentional:  _(could you double check it's okay?)_

**Comment by wen-2018** [comment]
> Oh that's lovely, I never noticed this animation before, so cool!
> 
> I've noticed something a bit off here, changing around the .py files formatting, that might have been unintentional: _(could you double check it's okay?)_

Yeah, these are all related to my editor auto formatting Python files on save. These are not intended unfortunately

**Comment by alexgibson** [comment]
@wen-2018 it might be worth double-checking your editor to try and disable any formatting tools that are not project-configured. I think most of the things we enforce _should_ be covered in our pre-commit hook.

**Comment by janbrasna** [comment]
@alexgibson Would you be interested in reverting these? (https://github.com/janbrasna/bedrock/commit/6175a47bd63a0afda2168cde189a232a0045a1c9 resolved a few conflicts and also kept the new translate_langs as-is due to its current line length not passing lining otherwise…)

**Pull request merged**
